datadog_checks_base/datadog_checks/utils/decorator.py

```diff
@@ -0,0 +1,17 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+from datadog_checks.config import is_affirmative
+from ddtrace import tracer, patch
+
+patch(requests=True)
```

Comment:
I know import order matters for ddtrace currently, but we need to make this condition too somehow.

Reply:
Hmm, let me think about that one.

Reply:
So it appears that the parent span has to be started in order for the underlying request to be patched and have an associated span be sent. Disabling the flag with the current state of the code submits 0 traces from the check.

Reply:
Let's move this to the `__init__.py` of the base package, it should be early enough.